### PR TITLE
Prac 5 code review request

### PR DESCRIPTION
**Pull request by bigdogmitchyp**
@Lucas-obrien

### CODE REVIEW

File: prac_05/emails.py [hunk 1]
```diff
@@ -6,6 +6,7 @@
 
 
 def main():
```
Comment (Lucas-obrien) [inline prac_05/emails.py:8]: Nice code! Have you though about splitting some of the main into more functions?

File: prac_05/wimbledon.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    print("Wimbledon Champions: ")
+    for name, count in champion_to_count.items():
+        print(name, count)
+    print(f"\nThese {len(countries)} countries have won Wimbledon: ")
+    print(", ".join(country for country in sorted(countries)))
+
+
+def process_records(records):
+    champion_to_count = {}
+    countries = set()
```
Comment (Lucas-obrien) [inline prac_05/wimbledon.py:27]: This feels a little ambiguous, could you specify what kind of countries you're listing here?

File: prac_05/emails.py [hunk 3]
```diff
@@ -20,6 +21,7 @@ def main():
 
 
 def get_name_from_email(email):
+    """Extract name from emails"""
     start = email.split("@")[0]
```
Comment (Lucas-obrien) [inline prac_05/emails.py:25]: Good splitting of the work here, clear to see whats happening at each line

File: prac_05/word_occurences.py [hunk 4]
```diff
@@ -0,0 +1,18 @@
+"""
+Cp1404 Practical
+estimate: 20 min
+actual: 18 min
+"""
+word_to_count = {}
+string = input("Text: ")
+words = string.split()
+for word in words:
+    frequency = word_to_count.get(word, 0)
```
Comment (Lucas-obrien) [inline prac_05/word_occurences.py:10]: Could you get this into one line?

File: prac_05/emails.py (hunk 3 above)
Comment (bigdogmitchyp) [reply prac_05/emails.py:25]: cheers

File: prac_05/wimbledon.py (hunk 2 above)
Comment (bigdogmitchyp) [reply prac_05/wimbledon.py:27]: thanks, had a look at it

File: prac_05/emails.py (hunk 1 above)
Comment (bigdogmitchyp) [reply prac_05/emails.py:8]: Cheers i refactored the name confirmation function

File: prac_05/word_occurences.py (hunk 4 above)
Comment (bigdogmitchyp) [reply prac_05/word_occurences.py:10]: i'm not sure that i can sorry

**Comment by bigdogmitchyp** [comment]
all good @Lucas-obrien?

**Pull request merged**